### DISCUSSION TITLE
On the Сamel sub, the bots constantly "hang" on the ramp leading to the reactor compartment.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#14834] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

On the camel, the bots constantly "hang" on the ramp leading to the reactor compartment.

Let's move the waypoints a little.
Near the fabricator, the bots often get stuck on the drain hatch. 
There, too, you need to move the waypoint a little.

### Reproduction steps

Give the bot the order to repair and after a short time you will find that it is trampling in the middle of the ramp. It is probably trying to reach the engine or the reactor from the ramp to repair it.

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been fixed in our private repo now, coming in a future unstable build.
